modules/social_features/social_tagging/social_tagging.module

```diff
@@ -558,3 +559,33 @@ function social_tagging_preprocess_profile(array &$variables) {
     }
   }
 }
+
+/**
+ * Implements hook_entity_form_display_alter().
+ */
+function social_tagging_entity_form_display_alter(EntityFormDisplayInterface $form_display, array $context) {
```

Comment:
@volodymyr-sydor-lemberg @ribel I am trying to understand why this function was removed in https://github.com/goalgorilla/open_social/pull/2450/files#diff-19de5cecbacf0e4c7d03f7e5f50073625b76b77849e19185b222a591a96e03dfL566 and then re-added in this?

Reply:
@navneet0693 
If we check the dates of merging those PR's/commits we can see that in current PR this function was added initially and in the second PR it was refactored/removed. I'm not sure why this function still exists, probably something went wrong during merging but now it should be removed.

Reply:
@volodymyr-sydor-lemberg I am also trying to find out the reason, from the git blame I get this :-/ https://github.com/goalgorilla/open_social/commit/620754e51b850437b327704d52e66796f820f6cb

Reply:
I think I found it, it got introduced in https://github.com/goalgorilla/open_social/pull/2313 probably due to branch merges.